### PR TITLE
Restored arrow key functionality to Menu component

### LINKED ISSUE #5935: Allow arrow key navigation in Menu
### Expected Behavior
You should be able to navigate in the Menu component with the up and down arrow keys.

### Actual Behavior
You can't navigate with arrow keys, you can only tab through the Menu.

### URL, screen shot, or Codepen exhibiting the issue
See Menu stories on storybook

### Steps to Reproduce
Try using arrow keys to navigate on a Menu

### PR DESCRIPTION
**Pull request by jcfilben**
#### What does this PR do?
Restores arrow key navigation to the Menu component

#### What testing has been done on this PR?
Tested with existing Menu stories

#### Do Jest tests follow these best practices?

- [ ] `screen` is used for querying.
- [ ] The correct query is used. (Refer to [this list of queries](https://testing-library.com/docs/queries/about/#priority))
- [ ] `userEvent` is used in place of `fireEvent`.
- [ ] `asFragment()` is used for snapshot testing.

#### What are the relevant issues?
Closes #5935

#### Do the grommet docs need to be updated?
No

#### Should this PR be mentioned in the release notes?
Yes

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible

### CODE REVIEW

**Comment by britt6612** [comment]
Screen Shot 2022-02-11 at 12 52 30 PM
can we get rid of this we may need to add something to the styled menu

**Comment by britt6612** [comment]
I know previously that `menu` opened with the `down arrow` this seems strange to me I would expect that  you can also hit enter this can be a separate PR but I wanted to get your thoughts

**Comment by jcfilben** [comment]
> can we get rid of this we may need to add something to the styled menu

Yes, I got rid of the outline in the latest commit

> I know previously that menu opened with the down arrow this seems strange to me I would expect that you can also hit enter this can be a separate PR but I wanted to get your thoughts

I agree, you should be able to open the select with 'enter'. I fixed this in the latest commit

File: src/js/components/Menu/Menu.js [hunk 1]
```diff
@@ -26,6 +28,10 @@ const ContainerBox = styled(Box)`
     width: 100%;
   }
```
Comment (britt6612) [inline src/js/components/Menu/Menu.js:30]: maybe just add a comment just a suggestion

**Comment by britt6612** [comment]
One small comment the rest looks good!

**Review by taysea — APPROVED** [review]

**Pull request merged**